tools/mo/openvino/tools/mo/convert.py

```diff
@@ -34,7 +34,7 @@ def convert_model(
         source_layout: [str, Layout, dict] = (),
         target_layout: [str, Layout, dict] = (),
         layout: [str, Layout, LayoutMap, list, dict] = (),
-        compress_to_fp16: bool = False,
+        compress_to_fp16: bool = True,
```

Comment:
Don't do that. We have `save_model` that is the place where we should apply fp16 compression.

Reply:
Agreed and reverted this line back